addons/mail/static/tests/helpers/mock_models.js

```diff
@@ -136,6 +136,8 @@ class MockModels {
                     res_model_name: { string: "Res Model Name", type: 'char' },
                     starred_partner_ids: { string: "Favorited By", type: 'many2many', relation: 'res.partner' },
                     subject: { string: "Subject", type: 'char' },
+                    subtype_description: { string: "Subtype description", type: 'char' },
```

Comment:
`mail.message.subtype` has a mock model, do we really need to create this fake field on message too? Ideally mock should return the proper value based on data in subtype itself.

Reply:
Didn't see it, my bad